src/bloom.cpp

```diff
@@ -23,15 +23,15 @@
 
 using namespace std;
 
-void CBloomFilter::setup(unsigned int nElements, double nFPRate, unsigned int nTweakIn, unsigned char nFlagsIn, bool size_constrained) {
+void CBloomFilter::setup(unsigned int nElements, double nFPRate, unsigned int nTweakIn, unsigned char nFlagsIn, bool size_constrained, uint32_t nMaxFilterSize) {
```

Comment:
format nit: bracket in the new line

Reply:
I'll have to rebase this anyway on top of #773  (the new formatting changes) when they're pulled in.

Reply:
ok